apps/nextjs-website/src/components/molecules/SubscribeToWebinar/SubscribeToWebinar.tsx

```diff
@@ -93,12 +93,23 @@ const SubscribeToWebinar = ({
       setIsLoading(false);
     });
     return null;
-  };
+  }, [
+    onUpdateAttributes,
+    router,
+    setIsSubscribed,
+    t,
+    webinarSlug,
+    webinarState,
+  ]);
 
   const onSubscribeWithoutUser = () => {
     setIsLoading(true);
+    const finalPath = !pathname.includes(`/webinars/${webinarSlug}`)
+      ? `/webinars/${webinarSlug}?action=subscribe`
+      : `${pathname}?action=subscribe`;
+
     // eslint-disable-next-line functional/immutable-data
-    router.push(`/auth/login?redirect=${pathname}`);
+    router.push(`/auth/login?redirect=${finalPath}`);
```

Comment:
The resulting URL is `/auth/login?redirect=/webinars/PagoPA-multe?action=subscribe` with a double question mark. You should replace `?` with `&` on line 109

Reply:
The second `?` is relative to the URL that the user will se after he has logged in. we can't change that with the `&`, one solution could be encode the redirectURL so the user don't see it in the first url